src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -161,25 +161,31 @@ p5.RendererGL.prototype._drawFillImmediateMode = function(
 ) {
   var gl = this.GL;
   this.curFillShader.bindShader();
-  //vertex position Attribute
-  this._bindBuffer(
-    this.immediateMode.vertexBuffer,
-    gl.ARRAY_BUFFER,
-    this._vToNArray(this.immediateMode.vertices),
-    Float32Array,
-    gl.DYNAMIC_DRAW
-  );
 
-  this.curFillShader.enableAttrib(
-    this.curFillShader.attributes.aPosition.location,
-    3,
-    gl.FLOAT,
-    false,
-    0,
-    0
-  );
+  if (this.curFillShader.attributes.aPosition) {
+    //vertex position Attribute
+    this._bindBuffer(
+      this.immediateMode.vertexBuffer,
+      gl.ARRAY_BUFFER,
+      this._vToNArray(this.immediateMode.vertices),
+      Float32Array,
+      gl.DYNAMIC_DRAW
+    );
 
-  if (this.drawMode === constants.FILL) {
+    this.curFillShader.enableAttrib(
+      this.curFillShader.attributes.aPosition.location,
+      3,
+      gl.FLOAT,
+      false,
+      0,
+      0
+    );
+  }
+
+  if (
```

Comment:
@lmccart any style guide suggestions here for multi-line if statement? I'm not sure we've used this convention elsewhere.

Reply:
#2336 added auto-formatting for the whole library. this is how it's configured, apparently.

Reply:
ok, we will go with what the auto-formatter wants :)